security/pkg/stsservice/tokenmanager/google/mock/mockserver.go

```diff
@@ -329,11 +329,9 @@ func (ms *AuthorizationServer) getAccessToken(w http.ResponseWriter, req *http.R
 		w.WriteHeader(http.StatusInternalServerError)
 		return
 	}
-	tokenLife.Format(time.RFC3339)
-	lifeJSON, _ := tokenLife.MarshalJSON()
 	resp := accessTokenResponse{
 		AccessToken: token,
-		ExpireTime:  string(lifeJSON),
+		ExpireTime:  tokenLife.Format(time.RFC3339Nano),
```

Comment:
isn't this changing from json? is that expected?

Reply:
I did search for that. One example of using JSON is like this
https://stackoverflow.com/a/23695774/8922031.
If we want to specify a time format when marshalling time, we need to create our helper function.
Please let me know if there is a better option.